data/data/bootstrap/gcp/files/usr/local/bin/report-progress.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+KUBECONFIG="${1}"
+
+wait_for_existance_file() {
+	while [ ! -e "${1}" ]
+	do
+		sleep 5
+	done
+}
+
+wait_for_existance_k8s_object() {
+	while ! oc --config="$KUBECONFIG" -n "${1}" get "${2}"
+	do
+		sleep 5
+	done
+}
+
+echo "Waiting for bootstrap to complete..."
+wait_for_existance_file /opt/openshift/.openshift.done
+wait_for_existance_k8s_object kube-system event/bootstrap-success
+
+echo "Reporting install progress..."
```

Comment:
What happens if the bootstrap instance gets torn down and the bootstrap pod is still not finished creating resources?

Reply:
> Relaxing the bootstrap-complete to openshift.service done and control-plane successfully running gives us the confidence that everything important is done.
But there's defintely a race where some assets maybe have not been pushed to the cluster which will be silently ignored, and since cluster-bootstrap is only responsible for pushing
objects required for bootstrapping control-plane this might be fine.

from the PR description...

Reply:
Ack... so if items fails to get created, they will somehow be created by operators?

Reply:
No, there non-existentence will be ignored

Reply:
And as I pointed out above, the chance of happening that is small enough for now due to higher-control of these that we might be fine....

And this is for gcp only, this can be temporary until we find solutions to reduce that gap..